packages/phone-number-privacy/migrations/20200330212301_create-number-pairs-table.ts

```diff
@@ -0,0 +1,12 @@
+import * as Knex from 'knex'
+
+export async function up(knex: Knex): Promise<any> {
+  return knex.schema.createTable('number_pairs', (t) => {
+    t.string('user_phone_hash').notNullable()
+    t.string('contact_phone_hash').notNullable()
```

Comment:
do we need to tell postgres to index these or is that automatic?

Reply:
I haven't addressed that yet, will do in a separate PR

Reply:
sounds good